src/Ombi.Api.Jellyfin/JellyfinApiFactory.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Ombi.Api;
+using Ombi.Core.Settings;
+using Ombi.Core.Settings.Models.External;
+using System.Threading.Tasks;
+
+namespace Ombi.Api.Jellyfin
+{
+    public class JellyfinApiFactory : IJellyfinApiFactory
+    {
+        private readonly ISettingsService<JellyfinSettings> _jellyfinSettings;
+        private readonly IApi _api;
+
+        // TODO, if we need to derive futher, need to rework
```

Comment:
```suggestion
        // TODO, if we need to derive further, need to rework
```

Reply:
A lot of this was straight copy+paste+replace, so a lot of these are not mine.

Reply:
That's fine. I'll sort it